app/code/community/Algolia/Algoliasearch/Model/Queue.php

```diff
@@ -7,6 +7,7 @@ class Algolia_Algoliasearch_Model_Queue
 
```

Comment:
File looks good to me, I'd like @JanPetr to double-check it to be 100% sure we didn't miss something ;)